persistence/migrations/2_add_invoice_id.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE lnmux.invoices ADD "sequence_num" SERIAL;
```

Comment:
UNIQUE constraint? And need to think about the upgrade path for existing invoices. Maybe just drop everything.